### ISSUE TITLE
Challenge IV

### ISSUE THREAD

**Issue by m00se333** [learn-co-curriculum/my-first-hash#7]
TL;DR - Asking a generic question and testing for a specific answer is not cool. 

The wording returned in the spec file is very misleading. I would suggest instead of it returning: "Challenge IV: Adding Data #adding operates on the shipping_manifest hash **to add a key/value pair**". Make it more specific. 

I tried adding the following key "guns" equal to 100000 or some crazy number, then I received this error: 

```
    operates on the shipping_manifest hash to add a key/value pair (FAILED - 1)

Failures:

  1) Challenge IV: Adding Data #adding operates on the shipping_manifest hash to add a key/value pair
     Failure/Error: expect(adding).to eq({"whale bone corsets" => 5, "porcelain vases" => 2, "oil paintings" => 3, "pearl necklace" => 1})

       expected: {"whale bone corsets"=>5, "porcelain vases"=>2, "oil paintings"=>3, "pearl necklace"=>1}
            got: {"whale bone corsets"=>5, "porcelain vases"=>2, "oil paintings"=>3, "guns"=>10000000}

       (compared using ==)

       Diff:

       @@ -1,5 +1,5 @@
       +"guns" => 10000000,
        "oil paintings" => 3,
       -"pearl necklace" => 1,
        "porcelain vases" => 2,
        "whale bone corsets" => 5,
```

I tried setting the value of "guns" to 1, and that returned a similar error. Then taking a que from the error that uses the specific example "pearl necklace" with a value of 1. I entered that into my code and the test passed. Just to be sure I then tried to add another value this time just "pearls" set to 2 and the test did not pass.

Please make what the spec file more true to what it is testing for. Simply telling someone "add a key/value pair" is misleading when you're looking for something specific.

**Comment by AnnJohn** [comment]
The readme indicates the specific key/value pair to add.
